### ISSUE TITLE
Fix Color Contrast Issues

### ISSUE THREAD

**Issue by teshsopher** [tl-its-umich-edu/remote-office-hours-queue#155]
There are some links and buttons that don't have enough contrast. Minimum contrast should be 4.5:1 (ideally 7:1 for WCAG AAA compliance).

Places this should be adjusted:
- Alert banners: Links are not a high enough contrast
- Breadcrumbs: Links in the breadcrumb don’t have high enough contrast
- Manage: “+ Add Queue button” does not have a sufficient contrast with the button text (Maybe a darker green?)
- Manage Queue: “+ Add Host” and “+ Add Attendee” buttons don’t have enough contrast (see previous point)
- Manage Queue: Disabled “Remove” button (trash icon) – Doesn’t have enough contrast; recommendation – Leave it enabled, but have an error message that tells why the user can’t click it instead
- Attendee View: “Manage it” link doesn’t have enough contrast

**Comment by ssciolla** [comment]
I'm going to take notes as I try to address these:

- [] "Alert banners: Links are not a high enough contrast" - 
- [] "Breadcrumbs: Links in the breadcrumb don’t have high enough contrast"
- [] "Manage: “+ Add Queue button” does not have a sufficient contrast with the button text (Maybe a darker green?)" -
- [] "Manage Queue: “+ Add Host” and “+ Add Attendee” buttons don’t have enough contrast (see previous point)" -
- [] "Manage Queue: Disabled “Remove” button (trash icon) – Doesn’t have enough contrast; recommendation – Leave it enabled, but have an error message that tells why the user can’t click it instead"
- [] "Attendee View: “Manage it” link doesn’t have enough contrast" -

**Comment by teshsopher** [comment]
Tested and working in QA
